src/Components/Navbar.js

```diff
@@ -0,0 +1,17 @@
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => (
+  <header>
+    <nav>
+      <ul>
+        <li>
+          <NavLink to="/">Books</NavLink>
+        </li>
+        <li>
+          <NavLink to="/">Categories</NavLink>
```

Comment:
- [ ] Navigation to the categories page does not really work. To match the routing in the App component, this should be:

  ```
  ...
  <NavLink to="/categories">Categories</NavLink>
  ...
  ```